profiling/nvtx-connector/kp_nvtx_connector.cpp

```diff
@@ -40,6 +40,24 @@ void kokkosp_init_library(const int loadSeq, const uint64_t interfaceVer,
   printf("KokkosP: NVTX Analyzer Connector (sequence is %d, version: %llu)\n",
          loadSeq, interfaceVer);
   printf("-----------------------------------------------------------\n");
+ const char* tool_global_fences = getenv("KOKKOS_TOOLS_NVTXCONNECTOR_GLOBALFENCES");
```

Comment:
We may actually just want to use KOKKOS_TOOLS_GLOBALFENCES?

Reply:
Changed to KOKKOS_TOOLS_GLOBALFENCES.